src/controllers/trust.involved.controller.ts

```diff
@@ -1,5 +1,5 @@
 import { NextFunction, Request, Response } from 'express';
-import { getTrustInvolvedPage, postTrustInvolvedPage } from "../utils/trust/trust.involved";
+import { getTrustInvolvedPage, postTrustInvolvedPage } from "../utils/trust.involved";
```

Comment:
How come a number of files have been brought up a directory, out of the trusts directory?

Reply:
I initially had the trust.involved file in the utils/trust folder, but after speaking with Cory took it out again and put it with the rest of the trusts files

Reply:
What's the reason? There are a number of files still in the /trust directory

Reply:
The trusts folder is specifically for holding trust related business logic thats used by the controllers, so we're separating it from our files. A separate folder for the trusts shared controller would be ideal though